pkg/types/kubevirt/platform.go

```diff
@@ -0,0 +1,24 @@
+package kubevirt
+
+// Platform stores all the global configuration that all
+// machinesets use.
+type Platform struct {
+	// The Namespace in the infra cluster, which the control plane (master vms)
+	// and the compute (worker vms) are installed in
+	Namespace string `json:"namespace"`
+
+	// The Storage Class used in the infra cluster
+	StorageClass string `json:"storageClass"`
+
+	// NetworkName is the target network of all the network interfaces of the nodes.
+	NetworkName string `json:"networkName"`
+
+	// APIVIP is an IP which will be served by bootstrap and then pivoted masters, using keepalived
+	APIVIP string `json:"apiVIP"`
+
+	// IngressIP is an external IP which routes to the default ingress controller.
+	IngressVIP string `json:"ingressVIP"`
+
+	// PersistentVolumeAccessMode is the access mode should be use with the persistent volumes
```

Comment:
The default value the installer use is `ReadWriteMany`.
It was decided that this parameter shouldn't be exposed to user, as in regular installation it shouldn't be the user decision.
Therefore this parameter is not exposed in the survey and we don't put the default value in the installconfig (we use here empty string as a default value, and than its not included in the file - omitempty)
In the Tfvars stage in the code, if the value is empty, we change it to `ReadWriteMany` (`safeAccessMode` in `pkg/tfvars/kubevirt/kubevirt.go`).
In case there is no other choise, and the user have to change it to `ReadWriteOnce`, the user can add the value to the installconfig file
Example why we need to keep this in the installconfig file:
When the infracluster installed in GCP (which is currently used for the CI), `ReadWriteMany` is not supported, therefore must to change it `ReadWriteOnce`

Reply:
Also relevant for this comment:
https://github.com/openshift/installer/pull/4350#discussion_r518311460

Reply:
I don't think that there is any precedence for this approach. If the field is in the install config type, then it is exposed to the user, regardless of whether it is omitted or not.

Reply:
Changed it - set the default here